cli.js

```diff
@@ -263,13 +263,29 @@ const commands = {
       console.log('Not implemented');
       doExit();
     } else {
-      execute(
-        'ps aux | grep "impress\\|%CPU" | grep -v "grep\\|status"',
-        () => {
-          console.log('Stopped');
-          doExit();
-        }
-      );
+      cp.exec('ps -e | awk \'{if ($4 == "impress") print $1}\'',
```

Comment:
It is mostly just a copy-paste from the `stop()` method above, I think it should be refactored into the separate function called something like `listImpressProcesses`.